lib/widgets/resources/base.dart

```diff
@@ -34,8 +34,8 @@ class _ViamBaseWidgetState extends State<ViamBaseWidget> {
 
   @override
   void initState() {
-    camera = widget.cameras.firstOrNull;
     super.initState();
+    camera = widget.cameras.firstOrNull;
```

Comment:
[q] why move this after the super init state? Was it getting overwritten or something?

Reply:
We've just been doing it wrong the entire time. From the [documentation](https://api.flutter.dev/flutter/widgets/State/initState.html):

> Implementations of this method should start with a call to the inherited method, as in super.initState().

Reply:
Similarly, you might see changes to `super.dispose()` in other PRs (not in this one) for the inverse reason:

> Implementations of this method should end with a call to the inherited method, as in super.dispose().

Reply:
Cool; makes intuitive sense too that you want to do the inherited construction/destruction before the custom class construction/destruction.